deployment/docker/provisioning/datasources/prometheus-datasource/cluster.yml

```diff
@@ -6,6 +6,10 @@ datasources:
       access: proxy
       url: http://vmauth:8427/select/0/prometheus
       isDefault: true
+      basicAuth: true
+      basicAuthUser: foo
+      secureJsonData:
+        basicAuthPassword: bar
```

Comment:
Could you also add a comment here and in vmauth config to check the other side if password is updated?